docs/aws-setup.md

```diff
@@ -1,7 +1,11 @@
-<!-- Target audience: 
-    - clinical researcher, low AWS familiarity
-    - engineer w/ some AWS experience
-reference tone -->
+---
+title: AWS Setup
+parent: Library
+nav_order: 2
+# audience: clinical researcher, low AWS familiarity or engineer w/ some AWS experience
+# type: reference
```

Comment:
I converted all the little audience & tone markers to audience & type comments here -- dropped all the "helpful direct" comments as the default value -- if we want to be casual or some other tone, we can say that explicitly.